### PR TITLE
Update teams-panels.md

### PR DESCRIPTION
**Pull request by lcansby**
Logitech added as certified panel.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/devices/teams-panels.md [hunk 1]
```diff
@@ -43,8 +43,11 @@ To learn more about these features and how to use them, see [Use Microsoft Teams
 You can acquire your Teams panels devices from one of the following partners:
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-panels.md:43]:
```suggestion
You can acquire your Teams panels devices from one of the following [currently certified Teams panels partners](/microsoftteams/devices/teams-ip-phones#currently-certified-teams-panels) partners:
```

File: Teams/devices/teams-panels.md [hunk 2]
```diff
@@ -43,8 +43,11 @@ To learn more about these features and how to use them, see [Use Microsoft Teams
...
 
 - Crestron
+- Logitech
 - Yealink
 
+[Currently certified Teams panels](https://docs.microsoft.com/en-us/microsoftteams/devices/teams-ip-phones#currently-certified-teams-panels)
+
```
Comment (JohanFreelancer9) [inline Teams/devices/teams-panels.md:50]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lcansby, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ManikaDhiman PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ManikaDhiman** [comment]
Hi @scanum, I am no longer working with the Teams panels doc team. Could you please reassign this PR to the appropriate author? Thanks!

**Comment by scanum** [comment]
Hi @ManikaDhiman Could you kindly help with providing the contact person? Thanks.

**Comment by scanum** [comment]
Hi @ManikaDhiman Could you kindly help with providing the contact person? Thanks.

**Comment by ManikaDhiman** [comment]
Hi @dstrome: Could you review the proposed changes please? Thanks!

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MSTonySmith** [comment]
Not approved by PM. Already in the list now.

**Pull request closed without merging**